docs/spec.yml

```diff
@@ -254,7 +367,100 @@ components:
         - scope
         - root_token_enabled
         - csrf_enabled
-  
+    Job:
+      type: object
+      properties:
+        id:
+          type: string
+        job_name:
+          type: string
+          description: Job Name
+        jenkins_server:
+          type: string
+          description: Jenkins Server
```

Comment:
At most, field descriptions copied from template - `job.js`

Reply:
That's reasonable. Descriptions can be tweaked later so I think that's good enough for now.